src/resources/wifi_esp32.cc

```diff
@@ -17,6 +17,7 @@
 
 #ifdef TOIT_FREERTOS
 
+#include <esp_heap_caps.h>
```

Comment:
Is this still needed?

Reply:
I don't even remember that I added it... Maybe an accidental "quick fix" from the IDE.
removed; done.